queries.sql

```diff
@@ -97,3 +97,48 @@ SELECT species, AVG(escape_attempts)
 FROM animals 
 WHERE YEAR(date_of_birth) BETWEEN 1990 AND 2000
 GROUP BY species;
+
+
+-- What animals belong to Melody Pond?
+SELECT a.name
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Melody Pond';
+
+-- List of all animals that are Pokemon (their type is Pokemon).
+SELECT a.name
+FROM animals a
+JOIN species s ON a.species_id = s.id
+WHERE s.name = 'Pokemon';
+
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT o.full_name, COALESCE(a.name, 'No animals owned') AS owned_animals
+FROM owners o
+LEFT JOIN animals a ON o.id = a.owner_id;
+
+-- How many animals are there per species?
+SELECT s.name AS species, COUNT(a.id) AS num_animals
+FROM species s
+LEFT JOIN animals a ON s.id = a.species_id
+GROUP BY s.name;
```

Comment:
**OPTIONAL** - Good job done so far, from the screenshot to your queries in your PR, the answer to this query is incorrect, I suggest you fix this issue to avoid incorrect answers in your subsequent milestone. A reason for this is that you have duplicate data in your database. `Angumon` and `Pikachu` have duplicated data. Also, from previous milestones `Ditto` should not be in your database. Please fix this 🙏